src/__tests__/pages/about-us/orbit-blog.test.tsx

```diff
@@ -91,8 +91,8 @@ describe('ORBIT Blog page', () => {
       jest.spyOn(useUserHooks, 'useUser').mockImplementation(() => {
         return {
           user: null,
-          portalUser:  null,
-          loading: true
+          portalUser: null,
+          loading: true,
```

Comment:
not sure what is happening with all of these formatted commas in files I did not even touch 🤨